spec/features/creating_a_journal_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
+
+RSpec.describe "Creating a journal" do
+
+  let(:admin) { FactoryGirl.create(:user, :administrator) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:facility) { FactoryGirl.create(:facility) }
+  let(:account) { FactoryGirl.create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: user), facility_id: facility.id) }
+  let!(:reviewed_order_detail) { place_and_complete_item_order(user, facility, account, true) }
+  let!(:unreviewed_order_detail) { place_and_complete_item_order(user, facility, account) }
+  let(:expiry_date) { Time.zone.now - 1.year }
+  let(:expired_payment_source) { FactoryGirl.create(:nufs_account, expires_at: expiry_date, account_users_attributes: account_users_attributes_hash(user: user), facility_id: facility.id) }
+  let!(:problem_order_detail) { place_and_complete_item_order(user, facility, expired_payment_source, true) }
+
+  before do
+    unreviewed_order_detail.update_attributes(reviewed_at: nil)
+    [reviewed_order_detail, problem_order_detail].each do |od|
+      od.update_attribute(:reviewed_at, 1.day.ago)
+    end
+
+    login_as admin
+  end
+
+  describe "new journal page" do
+    before do
+      visit new_facility_journal_path(facility)
+    end
+
+    it 'has only journalable order details' do
```

Comment:
This seems a little odd to call it "only journalable" when the page also has the un-journalable orders.